Add the fountain of youth

babydoll opened the pull request:
:cl: sabiram and Cakey
rscadd: The bird temple on the mining map has been redesigned and now houses the fountain of youth.
/:cl:

Assets courtesy of @CakeQ 
Sound courtesy of http://www.freesfx.co.uk/

also...
Makes the paper in bins not have writing on them
Adds modular computers to parrot perch list

babydoll committed:
Add the fountain of youth

Makes the paper in bins not have writing on them
Adds modular computers to parrot perch list

Sound courtesy of http://www.freesfx.co.uk/, modified by myself

chaoko99 commented:
While you're in there, could you slap an deny bag flag on paper bins? You can currently put them in your backpack and ruin your bag forever because you cannot remove it.

Maveriknight commented:
I can't help but wonder if it'd be possible to tweak scaling somehow. probably not, but it's a thought.

babydoll commented:
Not sure why the icon bot didn't run. 
![](https://i.imgur.com/ZmL7gfm.png)

babydoll committed:
Retype totem and fountain as base structures
Remap bird temple on mining-orb.dmm
Add light to fountain
Make parrot_grey.dmi darker
